FirebaseAuth/Sources/Swift/SystemService/AuthAppCredential.swift

```diff
@@ -15,7 +15,8 @@
 import Foundation
 
 /// A class represents a credential that proves the identity of the app.
-@objc(FIRAuthAppCredential) class AuthAppCredential: NSObject, NSSecureCoding {
+@objc(FIRAuthAppCredential) // objc Needed for decoding old versions
```

Comment:
Will this work for Firebase 10 to 11 transitions?

Reply:
Yes, see I didn't actually change anything about the type, I just added a newline to add the comment that objc is needed.

Reply:
I don't think the newline should interfere. There were some other instances in the SDK where this pattern was used. I thought it was good since it'd would be easy to mistakenly remove the objc without it.

Reply:
My question was more - can it still decode objects that were encoded with the NSObject class in Firebase 10?

Reply:
Oh, I understand. See that the NSObject superclass is unchanged. All I did here was add the comment.

Reply:
Oh, sorry - I see now. 😛